common/persistence/sql/sqlplugin/postgresql/visibility.go

```diff
@@ -318,5 +310,18 @@ func (mdb *db) GetFromVisibility(
 	if err != nil {
 		return nil, err
 	}
+	pdb.processRow(&row)
```

Comment:
If I understand correctly, this is the only functional change, and the rest of the code is refactoring, right?

Reply:
Yes. This is the actual fix. Refactoring the rest of the code to make them better, ie., less duplication.